sentinel-adapter/sentinel-motan-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/motan/SentinelMotanConsumerFilter.java

```diff
@@ -0,0 +1,71 @@
+package com.alibaba.csp.sentinel.adapter.motan;
+
+import com.alibaba.csp.sentinel.*;
+import com.alibaba.csp.sentinel.adapter.motan.config.MotanAdapterGlobalConfig;
+import com.alibaba.csp.sentinel.log.RecordLog;
+import com.alibaba.csp.sentinel.slots.block.BlockException;
+import com.weibo.api.motan.common.MotanConstants;
+import com.weibo.api.motan.core.extension.Activation;
+import com.weibo.api.motan.core.extension.SpiMeta;
+import com.weibo.api.motan.exception.MotanAbstractException;
+import com.weibo.api.motan.rpc.Caller;
+import com.weibo.api.motan.rpc.Request;
+import com.weibo.api.motan.rpc.Response;
+
+/**
+ * program: sentinel-parent
+ * description: ${description}
+ * author: zxn
+ * create: 2020-10-28 00:41
+ **/
+@Activation(key =MotanConstants.NODE_TYPE_REFERER)
+@SpiMeta(name = "sentinelMotanConsumer")
+public class SentinelMotanConsumerFilter extends BaseMotanSentinelFilter {
```

Comment:
It is suggested to add a space to the right of `=`, like `key = MotanConstants.NODE_TYPE_REFERER`, for better format.
Besides, create date can be removed to unified with other files.